logic/dungeon6.py

```diff
@@ -5,64 +5,181 @@
+            before_b_passage.connect(after_b_passage, r.boots_dash_2d, one_way=True) # boots dash over 1 block gaps in sidescroller
+            after_b_passage.connect(before_b_passage, AND(r.boots_dash_2d, r.boots_bonk), one_way=True) # boots dash over 1 block gaps in sidescroller, then bonk to get on ladder
+            before_c_passage.connect(after_c_passage, None) # damage_boost past the mini_thwomps
+            dodongo_room.connect(before_c_passage, r.boots_bonk) # boots bonk to escape dodongo room NE corner
+            #TODO: spark_pot_maze.connect(top_right_room, AND(POWER_BRACELET)) #[logic prep for staircase rando] # it's possible to pass through the room with only bracelet and not take damage, but it's hard
             
         if options.logic == 'glitched' or options.logic == 'hell':
-            elephants_heart_chest.connect(entrance, BOMB) # kill moldorm on screen above wizrobes, then bomb trigger on the right side to break elephant statue to get to the second chest
-            entrance.connect(left_side, AND(POWER_BRACELET, r.super_jump_feather), one_way=True) # path from entrance to left_side: use superjumps to pass raised blocks
-            lower_right_side.connect(center_2_and_upper_right_side, r.super_jump, one_way=True) # path from lower_right_side to center_2:  superjump from waterway towards dodongos. superjump next to corner block, so weapons added
-            center_1.connect(miniboss, AND(r.bomb_trigger, OR(r.boots_dash_2d, FEATHER))) # bomb trigger the elephant statue after the miniboss
-            center_2_and_upper_right_side.connect(center_1, AND(POWER_BRACELET, r.shaq_jump), one_way=True) # going backwards from dodongos, use a shaq jump to pass by keyblock at tile room
-            boss_key.connect(lower_right_side, AND(POWER_BRACELET, r.super_jump_feather)) # superjump from waterway to the left.
+            entrance.connect(second_elephant_room, BOMB) # kill moldorm on screen above wizrobes, then bomb trigger on the right side to break elephant statue to get to the second chest
+            wizrobe_switch_room.connect(south_star_area, r.super_jump_feather, one_way=True) # path from entrance to left_side: use superjumps to pass raised blocks
+            south_star_area.connect(star_area, AND(r.super_jump_feather, POWER_BRACELET)) # delayed superjump onto raised pegs so that you can pick up pot without switch hitter
+            before_c_passage.connect(dodongo_room, r.super_jump_feather, one_way=True) # superjump to get from pols to dodongos without kill requirements
+            after_miniboss.connect(before_b_passage, r.bomb_trigger) # go through north door exits to wrap-around, then bomb trigger as you transition into 2-statue room
+            flying_bomb_room.connect(after_b_passage, r.shaq_jump, one_way=True) # going backwards from dodongos, use a shaq jump to pass by keyblock at tile room

```

Comment:
maybe adjust comment to remove dodongo reference? That room is very far away from the defined locations now. Probably replace with horseheads or flying bomb room